RecoEgamma/EgammaTools/src/GBRForestTools.cc

```diff
@@ -0,0 +1,108 @@
+#include "RecoEgamma/EgammaTools/interface/GBRForestTools.h"
+
+#include <iostream>
+#include <fstream>
+
+// From https://stackoverflow.com/questions/18972258/index-of-nth-occurrence-of-the-string
+int strpos(std::string haystack, char needle, int nth)
+{// Will return position of n-th occurence of a char in a string.
+        std::string read;    // A string that will contain the read part of the haystack
+        for (int i=1 ; i<nth+1 ; ++i)
+        {
+                std::size_t found = haystack.find(needle);
+                read += haystack.substr(0,found+1); // the read part of the haystack is stocked in the read string
+                haystack.erase(0, found+1);     // remove the read part of the haystack up to the i-th needle
+                if (i == nth)
+                {
+                        return read.size();
+                }
+        }
+        return -1;
+}
+
+// To get the substring between the n1th and n2th quotation mark in a string
+std::string get_substring(std::string str, int n1, int n2)
+{
+    int pos = strpos(str, '"', n1);
+    int count = strpos(str, '"', n2) - pos;
+    return str.substr(pos, count - 1);
+}
+
```

Comment:
I would have stuck those functions in an anonymous namespace but up to you (eg namespace {  } )